azure/resources/base-az-resource/classes/base-az-resource-options.ps1

```diff
@@ -0,0 +1,16 @@
+Class BaseAzResourceOptions {
+    [string]$Name
+    [string]$SubscriptionName
+    [string]$ResourceGroupName
+    [hashtable]$Tags
```

Comment:
thinking probably worth scrubbing hashtables.  

i have some utility code i could pull in that can be used to set/get properties on psobject, and then can also iterate on the properties to mimic the list behavior.

Reply:
Yeah we could probably change it here, but it would have to be converted back to hastable before passing off to the az module to set the tags.